src/components/common/TargetCreationTab/index.js

```diff
@@ -0,0 +1,135 @@
+import useTranslation from 'hooks/useTranslation';
+import { useDispatch, useSelector } from 'react-redux';
+import { useGetTopicsQuery } from 'services/model/topics';
+import { setHomeTab } from 'state/slices/tabSlice';
+import { resetCreationTarget, fillCreationTarget } from 'state/slices/targetSlice';
+import { InputText } from 'primereact/inputtext';
+import { InputNumber } from 'primereact/inputnumber';
+import Back from '../Back';
+import { Dropdown } from 'primereact/dropdown';
+import { tabsEnum } from '../Tabs';
+import { Button } from 'primereact/button';
+import { selectTargets, useCreateTargetMutation } from 'services/model/targets';
+import Loader from '../Loader';
+import { useEffect, useCallback } from 'react';
+import { success, error } from 'react-toastify-redux';
+import './styles.scss';
+
+const TargetCreationTab = () => {
+  const dispatch = useDispatch();
+  const { data: topics } = useGetTopicsQuery();
+  const { creation } = useSelector(selectTargets);
+  const [createTarget, { isLoading, isSuccess, error: createError }] = useCreateTargetMutation();
+  const t = useTranslation();
+
+  const onBack = useCallback(() => {
+    dispatch(resetCreationTarget());
+    dispatch(setHomeTab(tabsEnum.intro));
+  }, [dispatch]);
+
+  useEffect(() => {
+    if (isSuccess || createError) {
+      if (isSuccess) dispatch(success(t('creationTab.success.created')));
```

Comment:
Is it necessary to check for `isSuccess` twice? in case you need to check both createError and isSuccess, maybe it's better to do an if / else if statement.

Reply:
You're right! It did sound weird. I can totally rework this.